examples/README.md

```diff
@@ -388,6 +388,104 @@ $ curl localhost:8000/v2/models/batching/stats
 {"model_stats":[{"name":"batching","version":"1","last_inference":1640111452223,"inference_count":2,"execution_count":1,"inference_stats":{"success":{"count":2,"ns":9997025869},"fail":{"count":0,"ns":0},"queue":{"count":2,"ns":9996491319},"compute_input":{"count":2,"ns":95288},"compute_infer":{"count":2,"ns":232202},"compute_output":{"count":2,"ns":195850}},"batch_stats":[{"batch_size":2,"compute_input":{"count":1,"ns":47644},"compute_infer":{"count":1,"ns":116101},"compute_output":{"count":1,"ns":97925}}]}]}
 ```
 
+### *BLS* Triton Backend
+
+The [*BLS*](backends/bls) backend demonstrates using in-process C-API to
+execute inferences within the backend. This backend serves as an example to
+backend developers for implementing their own custom pipeline in C++.
+For Python use cases, please refer to 
+[Business Logic Scripting](https://github.com/triton-inference-server/python_backend#business-logic-scripting)
+section in Python backend.
+
+The source code for the *bls* backend is contained in
+[backends/bls/src](backends/bls/src).
+
+* [backend.cc](backends/bls/src/backend.cc) contains the main backend
+implementation. The content of this file is not BLS specific. It only includes
+the required Triton backend functions that is standard for any backend
+implementation. The BLS logic is set off in the
+[`TRITONBACKEND_ModelInstanceExecute`](backends/bls/src/backend.cc#L317).
+function.
+
+* [bls.h](backends/bls/src/bls.h) is where the BLS (class `BLSExecutor`) of
+this example is located. You can refer to this file to see how to interact with
+Triton in-process C-API to build the custom execution pipeline.
+
+* [bls_utils.h](backends/bls/src/bls_utils.h) is where all the utilities that
+are not BLS dependent are located.
+
+The source code contains extensive documentation describing the operation of
+the backend and the use of the
+[Triton Backend API](../README.md#triton-backend-api) and the
+[Triton Server API](https://github.com/triton-inference-server/server/blob/main/docs/inference_protocols.md#in-process-triton-server-api).
+Before reading the source code, make sure you understand
+the concepts associated with Triton backend abstractions
+[TRITONBACKEND_Backend](../README.md#tritonbackend-backend),
+[TRITONBACKEND_Model](../README.md#tritonbackend-model), and
+[TRITONBACKEND_ModelInstance](../README.md#tritonbackend-modelinstance).
+
+The *bls* backend will send two requests on the 'addsub_python' and 'addsub_tf'
+models. After the inference requests are completed, this backend will extract
+OUTPUT0 from the 'addsub_python' and OUTPUT1 from the 'addsub_tf' model to
+construct the final inference response object using these tensors.
+
```

Comment:
Add a section here talking about the self-imposed limitations that were made for the simplicity of example.
Some of them that I see are: no batching, no decoupled models, no GPU tensors etc.

There might be some that I missed.

Reply:
Added the limitation of model configuration setting.